corgi/core/migrations/0096_load_missing_container_errata_builds.py

```diff
@@ -0,0 +1,60 @@
+from collections import defaultdict
+
+from django.db import migrations
+from django.db.models import Count
+
+from corgi.collectors.errata_tool import ErrataTool
+from corgi.tasks.errata_tool import slow_load_errata as current_slow_load_errata
+from corgi.tasks.prod_defs import brew_tag_to_et_prefixes
+
+
+def load_missing_container_errata_build(apps, schema_editor):
+    ProductStream = apps.get_model("core", "ProductStream")
+    CollectorErrataProductVersion = apps.get_model("collectors", "CollectorErrataProductVersion")
+
+    variants_by_stream = defaultdict(set)
+    # Allow this migration to be re-run in case of timeout or other failure
+    # Only get the streams we haven't processed yet
+    for stream in (
+        ProductStream.objects.annotate(num_components=Count("components"))
+        .filter(num_components__gt=0)
+        .exclude(tags__name="no_manifest")
```

Comment:
random observation: if this tag is some kind official 'minted' tag that we use in the system (for example collectors set this) ... then maybe we want to consider having a constant defined for this.

Reply:
Good idea, I created a constant in the tagging task.